optaweb-vehicle-routing-frontend/package-lock.json

```diff
@@ -1419,6 +1419,14 @@
             "supports-color": "^5.3.0"
           }
         },
+        "rimraf": {
+          "version": "2.7.1",
+          "resolved": "https://repository.engineering.redhat.com/nexus/repository/registry.npmjs.org/rimraf/-/rimraf-2.7.1.tgz",
```

Comment:
redhat proxy url; `locktt` been updated to replace these; see https://github.com/kiegroup/optaweb-employee-rostering/pull/320#issuecomment-552459818 (or just use sed)

Reply:
Good catch, thanks! Not sure if lock treatment tool will help me avoid this and #218 hasn't been merged yet. It seems I should really set up the commit hook to check this :slightly_smiling_face:.